src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -87,6 +87,11 @@ def _get_users() -> pd.DataFrame:
 result_bucket: Dict[str, pd.DataFrame] = {}
 
 
+# There are some methods that will need to share data in order to follow the
+# pattern used by the `_create_file_from_dataframe` method.
+shared_data: Dict = {}
```

Comment:
Why not just use the existing `result_bucket` variable? Can extend the type of the value instead of only typing it to `pd.DataFrame`. However, the name `shared_data` is better than `result_bucket` in that case.

Reply:
I might need to change the typing. Dataframe does not work for my functionality... If you are okay with it, I can use a single `bucket`

Reply:
I won't use the new shared data, just removed it